application/controllers/admin/surveyadmin.php

```diff
@@ -1183,6 +1186,9 @@ public function copy()
                 } elseif (!in_array(strtolower($sExtension), array('lss', 'txt', 'tsv', 'lsa'))) {
                     $aData['sErrorMessage'] = sprintf(gT("Import failed. You specified an invalid file type '%s'."), CHtml::encode($sExtension));
                     $aData['bFailed'] = true;
+                }elseif(!in_array($mimeType,['text/tab-separated-values','text/plain'])){
```

Comment:
Arg … unsure  on : 

- lss : `application/xml` or `text/xml`
- lsa : `application/zip`

?

Reply:
good question - not sure, either.

Reply:
Must test ;) with different server … :/

Reply:
Merged without resolving this?